pkg/notify/notify_test.go

```diff
@@ -31,8 +31,8 @@ func Test_getLatestReleaseTag(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			releaseTag, _ := getLatestReleaseTag()
-			if test.success == true {
+			releaseTag, err := getLatestReleaseTag()
```

Comment:
Can we mock this func instead using something like monkeypatching golang library and avoid genuine api call altogether? More info @ https://github.com/bouk/monkey

Why an actual get call in UT ?  I feel if you can mock the expected output tht would be more than sufficient in my opinion